set.mm

```diff
@@ -86206,6 +86246,19 @@ excluded middle (LEM), and in ILE the LEM is not accepted as necessarily
   php5 $p |- ( A e. _om -> -. A ~~ suc A ) $=
     ( com wcel csuc csdm wbr cen wn php4 sdomnen syl ) ABCAADZEFALGFHAIALJK $.
 
+  ${
+    php2d.1 $e |- ( ph -> A e. Fin ) $.
+    php2d.2 $e |- ( ph -> B C_ A ) $.
+    php2d.3 $e |- ( ph -> A ~~ B ) $.
+    $( Strengthening of ~ php expressed without negation.  (Contributed by
+       Rohan Ridenour, 3-Aug-2023.) $)
+    php2d $p |- ( ph -> A = B ) $=
```

Comment:
This is not the deduction form of ~php2, so it should be renamed. Maybe ~phpeqd. I also would change the comment to "Corollary of Pigeonhole Principle using equality. Strengthening of ~ php expressed without negation."

Reply:
Renamed to ` phpeqd ` and used your comment suggestion.